.circleci/config.yml

```
@@ -175,6 +175,21 @@ jobs:
             just prepare-json
             just simulate-council # simulate again to make sure the json is still valid
 
+  just_simulate_sep_019-fp-holocene-upgrade:
+    docker:
+      - image: <<pipeline.parameters.ci_builder_image>>
+    steps:
+      - checkout
+      - run:
+          name: just simulate sep_019-fp-holocene-upgrade
+          command: |
+            just install
+            cd tasks/sep/019-fp-holocene-upgrade
+            SIMULATE_WITHOUT_LEDGER=true just \
+             --dotenv-path $(pwd)/.env \
+             --justfile ../../../nested.just \
+             simulate foundation 0
```

in CI we should run it for both council and foundation, however if we get all sigs before this is merged we can change status to `[EXECUTED](https://txHashUrl)` and remove this task from CI
```suggestion
            just install
            cd tasks/sep/019-fp-holocene-upgrade
            SIMULATE_WITHOUT_LEDGER=true just \
             --dotenv-path $(pwd)/.env \
             --justfile ../../../nested.just \
             simulate foundation 0
            SIMULATE_WITHOUT_LEDGER=true just \
             --dotenv-path $(pwd)/.env \
             --justfile ../../../nested.just \
             simulate council 0
```